flowermd/base/system.py

```diff
@@ -731,21 +654,42 @@ def __init__(
         edge=0.2,
         overlap=0.2,
     ):
+        if not isinstance(density, u.array.unyt_quantity):
+            self.density = density * u.Unit("g") / u.Unit("cm**3")
+            warnings.warn(
+                "Units for density were not given, assuming "
+                "units of g/cm**3."
+            )
+        else:
+            self.density = density
         self.packing_expand_factor = packing_expand_factor
         self.edge = edge
         self.overlap = overlap
-        super(Pack, self).__init__(
-            molecules=molecules,
-            density=density,
-            base_units=base_units,
-        )
+        super(Pack, self).__init__(molecules=molecules, base_units=base_units)
 
     def _build_system(self):
-        self.set_target_box()
+        mass_density = u.Unit("kg") / u.Unit("m**3")
+        number_density = u.Unit("m**-3")
```

Comment:
Is there a reason we use `kg/cm^3` and `m^-3` in here instead of `g/cm^3` and `cm^-3`. specially since at the begining of this pack class we assume the default unit for density is`g/cm^3` if not specified.

Reply:
These variables are created so we can get the dimensions of the units instead of the units themselves.  We could use any unit of mass and length here, so I just went with the SI units.